concent_api/gatekeeper/views.py

```diff
@@ -120,6 +152,7 @@ def parse_headers(request: WSGIRequest, path_to_file: str) -> Union[FileTransfer
     except (MessageError, TypeError):
```

Comment:
Not necessarily a part of this task but it's a small change:

Can we still get `TypeError` here? If not, we should remove it.

Reply:
`TypeError` - no, but I think we can get `binascii.Error` from `b64decode`, so I will replace it.